### ISSUE TITLE
Bug when deserializing ?string 7.4-hinted properties.

### ISSUE THREAD

**Issue by signor-pedro** [doctrine/reflection#36]
Hi, anyone seeing a weird bug after composer update today?

If my column in DB is a nullable string and set to null, Doctrine does not hydrate the property with null but “ignores” the property altogether, which results in
```Typed property ... must not be accessed before initialization```
Started happening today after composer update.

![dbb](https://user-images.githubusercontent.com/53906348/77309922-bce6f600-6cfd-11ea-94d7-d6b1faff49b3.png)
See element 0 of array - it is missing event property altogether.
It did not happen yesterday, yesterday it was hydrated normally `null` as you would expect.

@beberlei

**Comment by marforon** [comment]
I have the same issue, I believe it's related to this https://github.com/doctrine/reflection/pull/35/commits/cd4811c644b8bcbf508367145573c5fd392b82a6.

**Comment by beberlei** [comment]
The property is declared as Nullable? Weird only non nullable properties should use the class, maybe i am wrong and its all typed classes, then we need to check for allowsnull

**Comment by signor-pedro** [comment]
Yes, it is declared as nullable, for reference:
```
/**
 * Fails
 * @ORM\Column(type="string", length=100, nullable=true)
 */
private ?string $event;

/**
* Works
* @ORM\Column(type="string", length=100, nullable=true)
*/
private $event;
```

**Comment by ondrejmirtes** [comment]
I agree, I have the same problem.

**Comment by beberlei** [comment]
Fixed by #37
